server/bin/docker/standalone_test.json

```diff
@@ -173,7 +173,7 @@
               "downsampleAggregator": "max",
               "downsampleFillPolicy": "none",
               "downsampleInterval": "1s",
-              "metric": "sys.cpu0.user",
+              "metric": "sys.cpu.user",
```

Comment:
I just saw that I haven't pushed this fix from my local box yet (you're welcome) Anyways, this file should be deleted and these changes should be made to grafana/timely-app/src/datasource/dashboards/standalone_test.json  The curl POST you added imports that file, not this one.